src/main.rs

```diff
@@ -29,6 +30,15 @@ fn main() -> ! {
         editor.load_history(path).ok();
     }
 
+    let mut vars: Vec<Variable> = Vec::new();
```

Comment:
1. Use the `vec!` macro
2. Use `String::from` for constant `String` values

```suggestion
    let mut vars = vec![
      Variable {
        repr: String::from("ans"),
        value: 0.0
      }
    ]
```

Reply:
Good call, much neater. Changed with 127dbe606ed8df144f0be1a1471a82e7dbdc8190